source/01-global/00-colors/color.scss

```diff
@@ -132,7 +132,7 @@ $easing-swatch: rem(75px);
   margin-block-end: 0.5rem;
   position: relative;
   transition-duration: 1s;
-  transition-property: left;
+  transition-property: inset-inline-start;
```

Comment:
Not really an issue for this PR but why do we have easing styles in color.scss in the first place? 🤔

Reply:
Looks like we (or maybe me?) accidentally left some of the code in the color file when splitting it into multiple files.  I've removed the unused code.